build/build.mjs

```diff
@@ -1,20 +1,18 @@
-'use strict';
-
-const fs = require('fs/promises');
-const path = require('path');
-const crypto = require('crypto');
-const minify = require('imagemin');
-const minifyPng = require('imagemin-pngquant');
-const minifyJpeg = require('imagemin-jpegtran');
-const fetch = require('node-fetch');
-const sharp = require('sharp');
-const Progress = require('./progress');
-const stringify = require('./stringify');
-const scraper = require('./scraper');
-const parser = require('./parser');
-const hashManager = require('./hashManager');
-const imageCache = require('../data/cache/.images.json');
-const exportCache = require('../data/cache/.export.json');
+import fs from 'fs/promises';
+import { createHash } from 'crypto';
+import minify from 'imagemin';
+import minifyPng from 'imagemin-pngquant';
+import minifyJpeg from 'imagemin-jpegtran';
+import fetch from 'node-fetch';
+import sharp from 'sharp';
+
+import Progress from './progress.mjs';
+import stringify from './stringify.mjs';
+import scraper from './scraper.mjs';
+import parser from './parser.mjs';
+import hashManager from './hashManager.mjs';
+import imagesCache from '../data/cache/.images.json' assert { type: 'json' };
```

Comment:
why'd this get renamed?

Reply:
looks like several got similar treatments. i'd rather not have name refactors that are hard to track on top of the language switchover

Reply:
it was an opsie that i made, _i blame vscode for that_